cli/run.go

```diff
@@ -610,3 +611,14 @@ func pad(s string, width int) string {
 	}
 	return s
 }
+
+func randomSessionID() string {
+	var b [8]byte
```

Comment:
16 bytes in base62 is still kind of short, might be worth never having to worry about it?